### PR TITLE
Merge with main

### PR DESCRIPTION
**Pull request by aludwiko**

### CODE REVIEW

File: sdk/spring-sdk/src/it/java/com/example/wiring/workflowentities/SpringWorkflowIntegrationTest.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.annotation.Import;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
```
Comment (octonato) [inline sdk/spring-sdk/src/it/java/com/example/wiring/workflowentities/SpringWorkflowIntegrationTest.java:29]: why that changed? 

probably `org.springframework.http.HttpStatus` is not in use (line above).

Comment (octonato) [reply sdk/spring-sdk/src/it/java/com/example/wiring/workflowentities/SpringWorkflowIntegrationTest.java:29]: Is that because of Spring 3.0.2?

Comment (aludwiko) [reply sdk/spring-sdk/src/it/java/com/example/wiring/workflowentities/SpringWorkflowIntegrationTest.java:29]: yes

**Review by octonato — APPROVED** [review]
LGTM

but we may have now an unused import.

**Pull request merged**